lib/pyszn/parser.py

```diff
@@ -52,8 +52,10 @@
 of a node and a port name.
 """
 
-from copy import deepcopy
+import glob
```

Comment:
I'd rather have you importing only what you need
```suggestion
from glob import glob
```

(I see you figured our weirdness of sorting imports by line length :) )

Reply:
I guess I'm weirdo cuz I always do the same